web/html/src/manager/content-management/shared/components/panels/build/build.js

```diff
@@ -75,7 +77,14 @@ const Build = ({projectId, onBuild, currentHistoryEntry = {}, changesToBuild, di
               content={
                 isLoading
                   ? <Loading text={t("Building project..")}/>
-                  :
+                      :
+                      <div>
+                      {hasChannelsWithUnsyncedPatches &&
+                      <Messages items={MsgUtils.info(
```

Comment:
Is an `info` type message the right one or? I am only wondering if a `warning` would make it more relevant or it is just a matter of user's choice.

Reply:
In this case I'd really prefer to stick to `info`, since we are not "warning" the user. We are "informing" them about this fact... Also a warning could shock them a bit :smile: